Lib/glyphsLib/builder/axes.py

```diff
@@ -282,20 +300,10 @@ def to_glyphs_axes(self):
         self.font.customParameters["Axes"] = axes_parameter
 
     if self.minimize_ufo_diffs:
-        # TODO: (jany) later, when Glyphs can manage general designspace axes
```

Comment:
Should we not set the Axis Mappings custom parameter in the UFO->Glyphs conversion unconditionally, instead of only when `minimize_ufo_diffs` is True? What's the reasoning behind not wanting to set that all the time?

Also I'm not sure if we should keep the TODO and the commented-out code below. cc @belluzj

Reply:
I agree that this should happen unconditionally, the idea behind "minimize_ufo_diffs" was to store in Glyphs some information that is only useful for minimizing diffs but that is not impacting functionality (like the order of keys in a dictionary for example) but this data about the axis mapping is not a detail; it's a core functionality of variable fonts, so it should be translated all the time.

Thank you very much for doing my TODOs :D yay for open-source!

Reply:
Ok, I've removed the conditional but I've also made sure that we only include the "Axis Mapping" customParameter if the designspace has map elements. This functionality is similar to when we convert a glyphs file to a designspace. It will only include map elements if they're actually useful.